week1/task_Judging the shape of a triangle.c

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h> 
+#include <math.h>
+#include <conio.h>
+
+#define EPS 1e-1
+
+int main ()
+{
+	while (1){ 

```

Comment:
Add a space before {
```suggestion
	while (1) {
```